auth-system/src/main/resources/application.yml

```
@@ -0,0 +1,3 @@
+spring:
+  application:
+    name: login-system
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)